test/TritonGPU/loop-pipeline.mlir

```diff
@@ -639,3 +639,215 @@ module attributes {"triton_gpu.compute-capability" = 80 : i32, "triton_gpu.num-c
     tt.return
   }
 }  // end module
+
+
+// -----
+
+// CHECK: #shared = #triton_gpu.shared<{vec = 4, perPhase = 1, maxPhase = 8, order = [1, 0], CTAsPerCGA = [1, 1], CTASplitNum = [1, 1], CTAOrder = [0, 1], hasLeadingOffset = false}>
```

Comment:
any chance this can be reduced or does it need all this to show the problem?

Reply:
Yeah, I further reduced the test. Please see if it's small enough. I feel it's going to be hard to reduce more.

Reply:
Thanks!